<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tareas</class>
 <widget class="QDialog" name="Tareas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>285</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>235</width>
    <height>284</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>285</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Agregar Tareas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Resources/bloc.png</normaloff>:/Resources/bloc.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nombre:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inNombre">
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>80</number>
     </property>
     <property name="placeholderText">
      <string>Ingrese el nombre de la tarea</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Fecha:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="inFecha"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Prioridad</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="inAlta">
        <property name="text">
         <string>Alta</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="inMedia">
        <property name="text">
         <string>Media</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="inBaja">
        <property name="text">
         <string>Baja</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Tipo:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inTipo">
     <property name="placeholderText">
      <string>Ejemplo: Universidad</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
